src/lib/updating-element.ts

```diff
@@ -200,29 +208,32 @@ export abstract class UpdatingElement extends HTMLElement {
   /**
    * Maps attribute names to properties; for example `foobar` attribute
    * to `fooBar` property.
+   * @nocollapse
    */
   private static _attributeToPropertyMap: AttributeMap = new Map();
 
   /**
    * Marks class as having finished creating properties.
    */
-  private static _finalized = true;
+  protected static finalized = true;
 
   /**
    * Memoized list of all class properties, including any superclass properties.
    */
-  private static _classProperties: PropertyDeclarationMap = new Map();
+  private static _classProperties?: PropertyDeclarationMap;
```

Comment:
why remove the initializer?

Reply:
`UpdatingElement` itself will never set anything into its `_classProperties` (since it never has `properties`); the set is created lazily during finalization on actual user subclasses

Reply:
Can add a comment to that effect.

Reply:
Done